package.json

```
@@ -30,6 +30,7 @@
     "jest-worker@^28.1.3": "patch:jest-worker@npm%3A28.1.3#./.yarn/patches/jest-worker-npm-28.1.3-5d0ff9006c.patch"
   },
   "dependencies": {
+    "@ethereumjs/tx": "^4.0.2",
```

```suggestion
    "@ethereumjs/tx": "^4.1.1",
```
Can we bump this to 4.1.1 please?  You actually have to do it in every module or you get:
```src/index.ts:117:9 - error TS2416: Property 'signTransaction' in type 'SimpleKeyring' is not assignable to the same property in base type 'Keyring<string[]>'.
  Type '(address: `0x${string}`, transaction: TypedTransaction, options?: Record<string, unknown>) => Promise<TxData>' is not assignable to type '(address: `0x${string}`, transaction: TypedTransaction, options?: Record<string, unknown> | undefined) => Promise<TxData>'.
    Types of parameters 'transaction' and 'transaction' are incompatible.```

---

[007206b](https://github.com/MetaMask/utils/pull/89/commits/007206b1acbf10abefecfd2cc59fb41844f302a6)